docs/qaoa/qaoa_ising.tst

```diff
@@ -0,0 +1,19 @@
+# Copyright 2021 The Cirq Developers
```

Comment:
2022

Reply:
wait... will this file be respected? I think this is part of the cirq testing framework

Reply:
Good catch, removed.